subsys/net/l2/Kconfig

```diff
@@ -72,6 +72,22 @@ source "subsys/net/l2/ppp/Kconfig"
 
 source "subsys/net/l2/ieee802154/Kconfig"
 
+config IEEE802154_NET_IF_NO_AUTO_START
+	bool "Disable 802.15.4 interface auto-start"
+	default n
```

Comment:
Option is disabled by default so setting it to 'n' can be omitted and we can remove this line.

Reply:
done